p22/tests/benchmark.rs

```diff
@@ -0,0 +1,18 @@
+#![feature(test)]
+
+extern crate test;
+
+#[cfg(test)]
+mod tests_benchmark_fibonacci {
```

Comment:
It's better to move these benchamrks to the benches folder. It also would allow you to keep the stable compiler in CI.

Reply:
applied those changes.

Reply:
thanks